### PR TITLE
add cve-2020-13945 vulnerability environment

### PR DESCRIPTION
**Pull request by hh-hunter**
@tooryx This is about the vulnerability environment submission for Apache APISIX(CVE-2020-13945).

### CODE REVIEW

**Comment by lokiuox** [comment]
Hey @hh-hunter , please implement the following changes:
- Rename the files according to the changes requested in [this comment](https://github.com/google/tsunami-security-scanner-plugins/pull/523#pullrequestreview-2236227030)
- Add an example safe configuration (e.g. with a non-default admin API key).

**Comment by hh-hunter** [comment]
Hey, I have fixed these problems, please review

File: apache/apisix/Default_api_token/docker-compose-non-vulnerable.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+version: "3.9"
```
Comment (lokiuox) [inline apache/apisix/Default_api_token/docker-compose-non-vulnerable.yml:1]:
```suggestion
```
Version is deprecated

File: apache/apisix/Default_api_token/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+version: "3.9"
```
Comment (lokiuox) [inline apache/apisix/Default_api_token/docker-compose.yml:1]:
```suggestion
```
Version is deprecated

File: apache/apisix/Default_api_token/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Apache APISIX Default API Token
+
+
+This directory contains the deployment configs for an Apache APISIX installation
+Apache APISIX has a built-in default API KEY. If the user does not proactively modify it (which few will), Lua scripts
+can be executed directly through the API interface, which can lead to RCE vulnerabilities.
+
+The deployed service has name `apache-apisix-defaul-api-token` and listens on port`9080`.
+
+The container that is not affected by the vulnerability is `apache-apisix-defaul-api-token-safe`,you can use it `docker-compose -f docker-compose-safe.yml up -d` to start the container, and the service listens on port `9081`.
```
Comment (lokiuox) [inline apache/apisix/Default_api_token/README.md:10]:
```suggestion
You can start the vulnerable service by running the command `docker compose up -d`. The deployed container has name `apache-apisix-defaul-api-token` and listens on port `9080`.

The container that is not affected by the vulnerability is `apache-apisix-defaul-api-token-safe`, you can start it with `docker compose -f docker-compose-safe.yml up -d`, and the service listens on port `9081`.
```
A small wording suggestion

**Review by lokiuox — CHANGES_REQUESTED** [review]
Hey @hh-hunter, here are a few small suggestions. Also please delete the old directory, i.e. `apache/apisix/CVE-2020-13945`. Besides these the testbed is fine.

**Review by lokiuox — APPROVED** [review]
Thanks, I think this is good to go.

**Comment by hh-hunter** [comment]
It's done. @lokiuox

**Review by tooryx — APPROVED** [review]

**Pull request merged**